lib/client.js

```diff
@@ -18,70 +18,110 @@ var GiantSwarm = (function () {
 
   return {
 
-    _authorizedRequest: function() {
-      if (authToken) {
-        return request.set('Authorization', 'giantswarm ' + authToken);
+    // to be thrown in case of bad parameters/arguments/values given
+    UserException: function(message) {
+      this.message = message;
```

Comment:
If it is only for internal use it should be good enough to just assume that message is always provided. Otherwise we should check that `message` is valid.